js/src/setup-mc/setup-stepper/setup-paid-ads/paid-ads-features-section.js

```diff
@@ -103,32 +94,34 @@ export default function PaidAdsFeaturesSection( {
 		>
 			<Section.Card>
 				<Section.Card.Body>
-					<Flex
-						className="gla-paid-ads-features-section__content"
-						align="center"
-						gap={ 9 }
-					>
-						<FlexBlock>
-							<Section.Card.Title>
-								{ __(
-									'Drive more sales with Performance Max',
-									'google-listings-and-ads'
-								) }
-							</Section.Card.Title>
-							<div className="gla-paid-ads-features-section__subtitle">
-								{ __(
-									'Reach more customers by advertising your products across Google Ads channels like Search, YouTube and Discover. Set up your campaign now so your products are included as soon as they’re approved.',
-									'google-listings-and-ads'
-								) }
-							</div>
-							<FeatureList
-								hideBudgetContent={ hideBudgetContent }
-							/>
-						</FlexBlock>
-						<FlexItem>
-							<CampaignPreview />
-						</FlexItem>
-					</Flex>
+					<VerticalGapLayout size="medium">
+						<Flex
+							className="gla-paid-ads-features-section__content"
+							align="center"
+							gap={ 9 }
+						>
+							<FlexBlock>
+								<Section.Card.Title>
+									{ __(
+										'Drive more sales with Performance Max',
+										'google-listings-and-ads'
+									) }
+								</Section.Card.Title>
+								<div className="gla-paid-ads-features-section__subtitle">
+									{ __(
+										'Reach more customers by advertising your products across Google Ads channels like Search, YouTube and Discover. Set up your campaign now so your products are included as soon as they’re approved.',
+										'google-listings-and-ads'
+									) }
+								</div>
+								<FeatureList />
+							</FlexBlock>
+							<FlexItem>
+								<CampaignPreview />
+							</FlexItem>
+						</Flex>
+
+						{ ! hideBudgetContent && <FreeAdCredit /> }
```

Comment:
This condition should be controlled by the `useFreeAdCredit` hook. Otherwise, it won't meet the acceptance criteria in #2499.

The reason for the previous use of the `hideBudgetContent` control was to comply with the requirement at that time: "Before the Google account is connected we can show the Boost card but we’ll want to limit the buttons and mentioning of Ads audience and budget." but not consider whether the account is eligible.

Reply:
The suggestion to use the `useFreeAdCredit()` was a part of the initial implementation brief, but is not part of the acceptance criteria. This was initially implemented using that hook, but we found that doing so didn't result in the expected visual behavior ([see related conversation](https://github.com/woocommerce/google-listings-and-ads/pull/2558#pullrequestreview-2261690968)).

I do think we need to clarify the correct behavior for determining when `useFreeAdCredit()` should return true, but would like to handle that in a separate issue. I'm fine with changing this PR back to using the hook in the meantime, which seems to be done in [90fd7e7](https://github.com/woocommerce/google-listings-and-ads/pull/2558/commits/90fd7e771ed1158b1b4a2d54981a27cf2b6b1710).

Reply:
> The suggestion to use the `useFreeAdCredit()` was a part of the initial implementation brief, but is not part of the acceptance criteria.

It's probably not about whether or not `useFreeAdCredit` was used. I meant that the actual results did not meet this acceptance criteria in #2499:

> * When an Ads account is not eligible, the component will not be shown.

With the 24c93287e63381abdf77640e6e071418407d14c5 revision, the banner is shown after connecting to an account that has been created more than 60 days ago.